glommio/src/sys/dma_buffer.rs

```diff
@@ -177,13 +177,7 @@ impl DmaBuffer {
     /// });
     /// ```
     pub fn read_at(&self, offset: usize, dst: &mut [u8]) -> usize {
-        if offset > self.size {
```

Comment:
for consistency, we could consider moving `write_at` to common code as well.
It's not a must since there is only one user currently.

Reply:
Huh, I think I've done that already?

Reply:
I must have missed it. Will look again, thanks!